### PR TITLE
Add tutorial to bridge SBY token with LayerZero

### PR DESCRIPTION
**Pull request by PierreOssun**

### CODE REVIEW

File: docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+:::info
+
+Note: The current method is provisional and specific to testnet environments. The eventual mainnet implementation will incorporate the bridging of Naive ASTR tokens through the Astar Portal.
+
+:::
+
+1. Fork the [Repository](https://github.com/AstarNetwork/layer-zero-bridge-contracts-testnet)
+
+2. Install dependencies by running `pnpm install`
```
Comment (andabak) [inline docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md:26]: Is this the only way? I don't have pnpm. Maybe some kind of comment 

Install dependencies by running 'pnpm install' or any other preferred package manager

File: docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+3. Create a .env file (based on the .env.example) in the root directory 
+
+4. Add your [private key](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key) in the .env file
+
+5. Run `pnpm hardhat compile` to compile the contracts
```
Comment (andabak) [inline docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md:32]: same as above npm vs pnpm

File: docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+#### ShibuyaZVM -> zKatana
```
Comment (andabak) [inline docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md:34]: This is a typo right Shibuya EVM (not ZVM)

File: docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+To send 1 SBY from shibuya-testnet to zkatana-testnet, run the following command (Note that the --quantity flag is in wei):
+
+`npx hardhat astarWithFeeSend --quantity 1000000000000000000 --target-network zkatana-testnet --network astar-testnet`
+
+#### zKatana -> ShibuyaZVM
```
Comment (andabak) [inline docs/build/zkEVM/integrations/bridges-relays/AstarEVM-zkEVM.md:39]: as above

**Review by meganskye — APPROVED** [review]
addressed your comments @andabak and polished. thank you for the submission @PierreOssun !

**Pull request merged**